spec/validator_spec.js

```diff
@@ -64,6 +64,14 @@ describe("XMLParser", function() {
         expect(result).toEqual(expected);
     });
 
+    it("should not consider this as self-closing tag", function() {
+        const xmlData = "<rootNode><in/valid></rootNode>";
+        const expected = {code: "InvalidAttr", msg: "attribute /valid has no space in starting."};
```

Comment:
Is this expected to be InvalidTag, and "Tag in/valid is an invalid name"? Or are you ok with this error?